### ISSUE TITLE
Feature idea: basic reference shapes for focal length matching

### ISSUE THREAD

**Issue by bpampuch** [stuffmatic/fSpy#119]
The only thing that I miss in fSPY is a set of tools which would allow me to match the focal length. 

Guidelines aren't accurate enough to find the right proportion.

Having some basic shapes (eg. a circle or a square) with the ability to rescale them uniformly would allow to go through the whole workflow (as presented in my tutorial https://youtu.be/daiMOYR8GS8?t=1013) directly in fSpy.

The current situation is:
- fSpy works perfectly in case of a 3-point perspective
- 1-point perspective is also quite simple, because we can easily determine the position of the principal point - therefore we can export the file, go to blender and find the focal length there,
- 2-point perspective can become horribly troublesome as the only tool that we have to influence the focal length is the horizontal position of the principal point (assuming both vanishing points are at the horizon, but this is the most common case)

Currently, the only way to do this (for 2-point perspective) is to:
1) guess the horizontal position of the PP
2) save the project
3) import it into Blender
4) match some basic shapes against the image
5) it they don't match go back to point 1 and repeat the process

Having the basic shapes directly in fSpy would make this workflow much simpler.

Two other features to better deal with 2-point perspective would be:
1) automatic placement of the principal point at the line between the vanishing points 
2) constraining the movement along this line

**Comment by BenRacicot** [comment]
Yes, I was trying to simply get the focal length also but found that without knowing the camera sensor size (in mm) we get `Image/sensor proportions mismatch`. I

 wonder if there could be some way to either auto detect the sensor or perhaps just updating the list of cameras in the dropdown would help with that.

**Comment by bobbbobobb** [comment]
I echo the above. 
it's a really useful bit of software, but it's lacking that control of 3rd VP and the sensor mismatch means what?

**Comment by BenRacicot** [comment]
Is it just me guys, or could NeRFs, Gaussian splats and general 3D figure out environmental math with the focal length instead of a requirement for multiple images.
